sds_in_a_box/sds_in_a_box_stack.py

```diff
@@ -72,6 +73,17 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             )
         )
 
+        # add an access policy for opensearch
+        domain.add_access_policies(
```

Comment:
Is this access policy too unrestricted? There needs to be an access policy for the lambda to access opensearch, I got it working with this policy, but I'm not sure if this is too open for our purposes.

Reply:
This link has some good examples of IAM policy: https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ac.html

Reply:
The following issue was opened to address this comment
#22